e2e/multi-platform.spec.ts

```diff
@@ -64,7 +68,7 @@ const cleanListAfterBuild = new Set(
   ),
 );
 
-test.describe(`multi platform builds`, () => {
+test.describe.serial(`multi platform builds`, () => {
```

Comment:
Is `.serial` required because we use workers=3 in playwright config?

Reply:
I think `fullyParallel: true` is the aggressive one. This even parallelize all tests inside a single test file, which in this case tries to run multiple builds on the same directory. This is somehow "working" in current Waku, but my RSC plugin breaks.

Reply:
@himself65 Any thoughts on it?